### DISCUSSION TITLE
New features & improvement requests

### DISCUSSION THREAD

**Discussion by LiuCMU** [isayevlab/Auto3D_pkg#65] [Ideas]
- [x] Accept SDF as the input
- [x] Using Multiple GPUs
- [x] Function API
- [x] Swap date and name in the folder name
- [x] handle IDs with underscore
- [ ] Detailed log information by logging in multiple processes
- [x] run from terminal without calling python. For example, type `auto3d input.smi --k=1`. Thanks @j3mdamas for implementing the entry point feature!
- [x] remove unnecessary variables `atom_list` and `num_atoms` [here](https://github.com/isayevlab/Auto3D_pkg/blob/3e685283db73c89d1fb87f141042d6803564fe49/src/Auto3D/isomer_engine.py#L156)
- [ ] new enantiomer [check](https://github.com/rdkit/rdkit/discussions/7169)
- [ ] different duplicate threshold for cyclic and non-cyclic molecules
- [x] accepts custom NNP
- [ ] rename output `E_tot` to `SPE_hartree`
- [ ] For each Jupyter Notebook documentation in readme.io, state the file link to GitHub

**Comment by CurtisColwell** [comment]
It would be nice to be able to filter out mirror image conformers prior to energy minimization for a speed boost.
